clients/sample-app/src/lib/db.ts

```diff
@@ -1,18 +1,13 @@
 import { Pool } from 'pg';
 
 const { DATABASE_HOST, DATABASE_PORT, DATABASE_USER, DATABASE_PASSWORD, DATABASE_DB } = process.env;
-if (!DATABASE_HOST) throw new Error('No DATABASE_HOST');
-if (!DATABASE_PORT) throw new Error('No DATABASE_PORT');
-if (!DATABASE_USER) throw new Error('No DATABASE_USER');
-if (!DATABASE_PASSWORD) throw new Error('No DATABASE_PASSWORD');
-if (!DATABASE_DB) throw new Error('No DATABASE_DB');
```

Comment:
I moved defaults for all of these just straight into the app. This removes the need to use the Dockerfile to spin this up and use it, though it'll basically just not work if you don't have a DB running!

...but it'll build & lint which is what I wanted!